main.go

```diff
@@ -35,36 +35,41 @@ type Input struct {
 	UsernameOnError       string          `env:"from_username_on_error"`
 	ThreadTs              string          `env:"thread_ts"`
 	ThreadTsOnError       string          `env:"thread_ts_on_error"`
+	Ts                    string          `env:"ts"`
+	TsOnError             string          `env:"ts_on_error"`
 	ReplyBroadcast        bool            `env:"reply_broadcast,opt[yes,no]"`
 	ReplyBroadcastOnError bool            `env:"reply_broadcast_on_error,opt[yes,no]"`
 
 	// Attachment
-	Color           string `env:"color,required"`
-	ColorOnError    string `env:"color_on_error"`
-	PreText         string `env:"pretext"`
-	PreTextOnError  string `env:"pretext_on_error"`
-	AuthorName      string `env:"author_name"`
-	Title           string `env:"title"`
-	TitleOnError    string `env:"title_on_error"`
-	TitleLink       string `env:"title_link"`
-	Message         string `env:"message"`
-	MessageOnError  string `env:"message_on_error"`
-	ImageURL        string `env:"image_url"`
-	ImageURLOnError string `env:"image_url_on_error"`
-	ThumbURL        string `env:"thumb_url"`
-	ThumbURLOnError string `env:"thumb_url_on_error"`
-	Footer          string `env:"footer"`
-	FooterIcon      string `env:"footer_icon"`
-	TimeStamp       bool   `env:"timestamp,opt[yes,no]"`
-	Fields          string `env:"fields"`
-	Buttons         string `env:"buttons"`
+	Color             string `env:"color,required"`
+	ColorOnError      string `env:"color_on_error"`
+	PreText           string `env:"pretext"`
+	PreTextOnError    string `env:"pretext_on_error"`
+	AuthorName        string `env:"author_name"`
+	Title             string `env:"title"`
+	TitleOnError      string `env:"title_on_error"`
+	TitleLink         string `env:"title_link"`
+	Message           string `env:"message"`
+	MessageOnError    string `env:"message_on_error"`
+	ImageURL          string `env:"image_url"`
+	ImageURLOnError   string `env:"image_url_on_error"`
+	ThumbURL          string `env:"thumb_url"`
+	ThumbURLOnError   string `env:"thumb_url_on_error"`
+	Footer            string `env:"footer"`
+	FooterIcon        string `env:"footer_icon"`
+	FooterOnError     string `env:"footer_on_error"`
+	FooterIconOnError string `env:"footer_icon_on_error"`
```

Comment:
I suggest following the existing pattern, that the `OnError` version of the input is right after its regular version.

```suggestion
	Footer            string `env:"footer"`
	FooterOnError     string `env:"footer_on_error"`
	FooterIcon        string `env:"footer_icon"`
	FooterIconOnError string `env:"footer_icon_on_error"`
```

Reply:
Fixed